modular_skyrat/master_files/code/modules/mob/living/simple_animal/friendly/dogs.dm

```diff
@@ -0,0 +1,124 @@
+/mob/living/simple_animal/pet/dog/markus
+	name = "\proper Markus"
+	real_name = "Markus"
+	gender = MALE
+	desc = "It's the Cargo's overfed, yet still beloved dog."
+	icon = 'modular_skyrat/master_files/icons/mob/pets.dmi'
+	icon_state = "markus"
+	icon_dead = "markus_dead"
+	icon_living = "markus"
+	speak = list("Borf!", "Boof!", "Bork!", "Bowwow!", "Burg?")
+	butcher_results = list(/obj/item/food/burger/cheese = 1, /obj/item/food/meat/slab = 2, /obj/item/trash/syndi_cakes = 1)
+	animal_species = /mob/living/simple_animal/pet/dog
+	can_be_held = FALSE
+	gold_core_spawnable = FRIENDLY_SPAWN
+
+/mob/living/simple_animal/pet/dog/markus/treat_message(message)
+	return client ? pick(speak) : message //markus only talks business
+
+/datum/chemical_reaction/mark_reaction
+	results = list(/datum/reagent/liquidgibs = 15)
+	required_reagents = list(/datum/reagent/blood = 20,
+	/datum/reagent/medicine/omnizine = 20,
+	/datum/reagent/medicine/c2/synthflesh = 20,
+	/datum/reagent/consumable/nutriment/protein = 10,
+	/datum/reagent/consumable/nutriment = 10,
+	/datum/reagent/consumable/ketchup = 5,
+	/datum/reagent/consumable/mayonnaise = 5,
+	/datum/reagent/colorful_reagent/powder/yellow/crayon = 5)
+	required_catalysts = list(/datum/reagent/consumable/enzyme = 5)
+	required_temp = 480
+
+/datum/chemical_reaction/mark_reaction/on_reaction(datum/reagents/holder, datum/equilibrium/reaction, created_volume)
+	. = ..()
+	var/location = get_turf(holder.my_atom)
+	new /mob/living/simple_animal/pet/dog/markus(location)
+	playsound(location, 'modular_skyrat/master_files/sound/effects/dorime.ogg', 100, 0, 7)
+
+/mob/living/simple_animal/pet/dog/corgi/borgi
+	name = "E-N"
+	real_name = "E-N" //Intended to hold the name without altering it.
+	desc = "It's a borgi."
+	icon = 'modular_skyrat/master_files/icons/mob/pets.dmi'
+	icon_state = "borgi"
+	icon_living = "borgi"
+	icon_dead = "borgi_dead"
+	maxHealth = 80
+	health = 80
+	var/emagged = 0
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	loot = list(/obj/effect/decal/cleanable/oil/slippery)
+	butcher_results = list(/obj/item/clothing/head/corgi/en = 1, /obj/item/clothing/suit/corgisuit/en = 1)
+	deathmessage = "its mechanics hiss before seizing."
```

Comment:
That's bound to look a little weird in chat, something like this might be better?
```suggestion
	deathmessage = "beeps, its mechanics hissing before its chassis collapses in a loud thud."
```
Also, I didn't suggest it because I didn't verify, but there's procs that are something like `p_his()` or something like that that give the proper pronoun for you based on the gender.

Reply:
i dont believe there is a way to give a pet the 'it' gender, so im leaving it as your recommendation for now.

Reply:
Genderless is the define NEUTER

Reply:
Genderless is the define NEUTER